<launch>
 <!-- Start the planning node -->
 <node name="sequential_planning" pkg="graph_search" type="start_and_goal" output="screen" required="true">
    <rosparam command="load" file="$(find graph_search)/configs/map.yaml"/>
 </node>

<!-- Start the map publishing node -->
 <node name="map_publish" pkg="graph_search" type="mappublish" output="screen">
 </node>

<!-- Start RVIZ for visualization -->
<!-- 在rviz中显示-->
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find graph_search)/configs/planning_visualization.rviz" required="true" />


</launch>